src/nwb_benchmarks/command_line_interface.py

```diff
@@ -115,10 +124,15 @@ def main() -> None:
                 raw_environment_info_file_path=raw_environment_info_file_path,
             )
             upload_results()
+        clean_cache(ignore_errors=True)
     elif command == "upload":
         upload_results()
     elif command == "clean":
         clean_results()
+        clean_cache(ignore_errors=False)
+    elif command == "config_set_cache":
+        cache_directory = pathlib.Path(sys.argv[2])
+        set_cache_directory(cache_directory=cache_directory)
```

Comment:
This is also a nice feature that will let us test behavior across different drives